lib/stream/bundle.js

```diff
@@ -99,6 +99,11 @@ function bundle(data){
 	// Combine all bundles
 	var allBundles = splitMainBundles.concat(splitBundles);
 	allBundles = allBundles.concat.apply(allBundles, unbundledBundles);
+
+	// in each multibuild we will start from beginning.
+	// with that, it is possible now, that file collisions are present
+	// in different multibuilds
+	nameBundle.clearBundleCounter();
```

Comment:
@matthewp i added this function, because if not, the tests will fail.
the problem is that the tests run mutliple build in one task. currently the `usedBundleNames` is leaking.

there are 2 problems with `usedBundleNames`

* if its not leaking, we cant cover file collisions with multiple builds in one task
* if it is leaking and we remove the `nameBundle.clearBundleCounter();` the tests will fail (https://github.com/stealjs/steal-tools/blob/master/test/multibuild_test.js#L2004) because the test to not check the `bundleCounter`